CONTRIBUTING.md

```diff
@@ -2,7 +2,8 @@
 
 We welcome contributions of any size and skill level. As an open source project, we believe in giving back to our contributors and are happy to help with guidance on PRs, technical writing, and turning any feature idea into a reality.
 
-> [!Tip] > **For new contributors:** Take a look at [https://github.com/firstcontributions/first-contributions](https://github.com/firstcontributions/first-contributions) for helpful information on contributing
+> [!Tip]
+> **For new contributors:** Take a look at [https://github.com/firstcontributions/first-contributions](https://github.com/firstcontributions/first-contributions) for helpful information on contributing
```

Comment:
Seems like our formatter was inlining it like before, which it would also happen here after we merge due to our auto formatter.

I think we can inject another `>` to prevent that, which seems to still render correctly.
```suggestion
> [!Tip]
>
> **For new contributors:** Take a look at [https://github.com/firstcontributions/first-contributions](https://github.com/firstcontributions/first-contributions) for helpful information on contributing
```

Reply:
Ohh I see.
It looks good just for now👍